src/pages/basic_data/app_admin/central_import/CentralImportDuplicationCheckPage.tsx

```diff
@@ -10,10 +10,12 @@ import { PageSpinner } from '../../../../components/PageSpinner';
 import { stringIncludesMathJax, typesetMathJax } from '../../../../utils/mathJaxHelpers';
 import { CentralImportDuplicateSearch } from './CentralImportDuplicateSearch';
 import NotFound from '../../../errorpages/NotFound';
+import { DuplicateSearchFilterForm, emptyDuplicateSearchFilter } from './DuplicateSearchFilterForm';
 
 export const CentralImportDuplicationCheckPage = () => {
   const { t } = useTranslation('basicData');
   const { identifier } = useParams<{ identifier: string }>();
+  const [duplicateSearchFilters, setDuplicateSearchFilters] = useState(emptyDuplicateSearchFilter);
```

Comment:
Spørsmål: vil dette virke likedan om du flytter den til under henting av `registration` og fjerner `useEffect`?
```suggestion
  const [duplicateSearchFilters, setDuplicateSearchFilters] = useState({...emptyDuplicateSearchFilter, doi:registration?.entityDescription?.reference?.doi ?? ''});
```

Reply:
shit. den klarte jeg ikke tenke på. Er vant til å tenke på axios-kall som asynkrone, og man må ha alt i useeffect.